scipy/spatial/transform/_rotation.pyx

```diff
@@ -2320,13 +2367,93 @@ cdef class Rotation:
         cdef double[:] angles = _empty1(num_rotations)
 
         for ind in range(num_rotations):
-            angles[ind] = 2 * atan2(_norm3(quat[ind, :3]), abs(quat[ind, 3]))
+            angles[ind] = _quat_magnitude(quat[ind])
 
         if self._single:
             return angles[0]
         else:
             return np.asarray(angles)
 
+    @cython.embedsignature(True)
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    def angle_to(Rotation self, Rotation other):
+        """Get the smallest angular distance(s) to another rotation.
+
+        For two rotations ``p`` and ``q``, the angular distance
+        ``p.angle_to(q)`` is equivalent to calculating
+        ``(q * p.inv()).magnitude()``.
+
+        Parameters
+        ----------
+        other : `Rotation` instance
+            Object containing the rotations to measure the angle to.
+
+        Returns
+        -------
+        angle : ndarray or float
+            Angle(s) in radians, float if object contains a single rotation
+            and ndarray if object contains multiple rotations. The angle
+            will always be in the range [0, pi].
+
+        Examples
+        --------
+        >>> from scipy.spatial.transform import Rotation as R
+        >>> import numpy as np
+        >>> p = R.from_quat([0, 0, 0, 1])
+        >>> q = R.from_quat(np.eye(4))
+        >>> p.angle_to(q)
+        array([3.14159265, 3.14159265, 3.14159265, 0.        ])
+
+        Angle to a single rotation:
+
+        >>> p.angle_to(q[0])
+        3.141592653589793
+        """
+        return self.rotation_to(other).magnitude()
+
+
+    @cython.embedsignature(True)
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    def approx_equal(Rotation self, Rotation other, double atol=1e-8):
```

Comment:
Similarly to `from_euler, from_rotvec` we should provide `degrees=False` flag to allow comparison with `atol` given in degrees. 

In order to fix the default tolerance irrespective of `degrees` flag, it can be changed to `None` and interpreted as `1e-8` radians no matter if `degrees` is True or False.

@scottshambaugh what do you think?

Reply:
Makes sense to me, updated!